tools/mappings/NIST SSDF v1.1 to NIST CSF 2.0/ssdf_1.1_to_csf_2.0.py

```
@@ -0,0 +1,21 @@
+import pandas as pd
+
+# Charger la feuille "Relationships", ignorer la première ligne
+df = pd.read_excel("./SSDF_to_CSF_2_0_0_CROSSWALK.xlsx", sheet_name="Relationships", skiprows=1, header=None)
+
+associations = []
+
+for index, row in df.iterrows():
```

Ruff (0.8.2)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)